internal/syncer/syncer.go

```diff
@@ -102,15 +102,15 @@ func (w *worker) exec(ctx context.Context, id string) {
 					continue
 				}
 
-				w.logger.Err(err).Msgf("error dequeuing job: %v", err)
+				w.logger.Warn().AnErr("error", err).Msgf("error dequeuing job: %v", err)
```

Comment:
last one - I think this one should be an `.Err(...)` call as well, we'll want to know why a job couldn't get dequeued